### ISSUE TITLE
Apply for an RR Type

### ISSUE THREAD

**Issue by moonshiner** [each/draft-aname#48]
I think we can apply for an RR Type now with IANA and go through expert review.  If there is no extra processing it can move forward with a published draft. 

glad to work on this.

**Comment by each** [comment]
... but there is extra processing, isn't there?

**Comment by matje** [comment]
Yes there is: When an A or AAAA query comes in, if there is an ANAME on the same domain, we would like to add it to the addition section. Also when an ANAME query comes in we would like to add A and AAAA in the additional section.

Also, although it is optional, a resolver may trigger additional queries when it retrieves an ANAME.

**Comment by Habbie** [comment]
This was one of the reasons splitting into separate auth and resolver documents was proposed some IETFs ago - a compact auth-only document (with a 'provisioning' focus perhaps) would allow the early RRtype allocation because it would not require special processing in all name servers. As the draft stands today, I don't think we'd get away with this (but I'd love to try!).

**Comment by moonshiner** [comment]
moonshiner  agree with Habbie on this.  I think splitting it up will make the RR Type bit go smoother.
Also, current implementations don't consider resolvers.

**Comment by wtoorop** [comment]
Would be a real pity to not facilitate resolver anticipation (and cater for offline signers or non-signing secondaries).  It is the whole reason for this draft to be like it is now, right?

**Comment by wtoorop** [comment]
So, to be clear, /me votes to not split.  The additional processing is not a big deal and caters for deployments, that /me would hate to lose/disregard.

**Comment by matje** [comment]
Faster implementation and faster document progress was for me the reason to suggest splitting up the document, but I did not hear strong support for that in the WG (to me the arguments made were leaning towards iterate on the draft as is).

**Comment by Habbie** [comment]
> Would be a real pity to not facilitate resolver anticipation (and cater for offline signers or non-signing secondaries). It is the whole reason for this draft to be like it is now, right?

Not to me, but it looks like I'm a minority.

**Comment by moonshiner** [comment]
My reasoning on keeping the resolver conversations separate was firmly in the focus of speedier adoption by the WG *and* the early RRType

I want to balance the forward thinking of resolver anticipation  with actually working consensus.
